### ISSUE TITLE
App crash after r593 on Android 8.1.0

### ISSUE THREAD

**Issue by kongwoojin** [chenxiaolong/DualBootPatcher#1165]
After r593, App crash in Roms menu, And my device is Nexus 6P
Logs: 
```
06-02 11:54:31.374 18268 18268 E AndroidRuntime: android.app.RemoteServiceException: Bad notification for startForeground: java.lang.RuntimeException: invalid channel for service notification: Notification(channel=null pri=-2 contentView=null vibrate=null sound=null defaults=0x0 flags=0x40 color=0x00000000 category=service vis=PRIVATE)
06-02 11:54:31.374 18268 18268 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1768)
06-02 11:54:31.374 18268 18268 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
06-02 11:54:31.374 18268 18268 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:164)
06-02 11:54:31.374 18268 18268 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:6494)
06-02 11:54:31.374 18268 18268 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-02 11:54:31.374 18268 18268 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
06-02 11:54:31.374 18268 18268 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
06-02 11:54:31.374 18268 18268 E MainApplication: Thread[main,5,main] has crashed with an uncaught exception; dumping log
```

**Comment by fxsheep** [comment]
Same

**Comment by chenxiaolong** [comment]
This should be fixed in `9.3.0.r640.gb6510967`
